taskcluster/ci/push-addon/kind.yml

```diff
@@ -0,0 +1,45 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - mozillavpn_taskgraph.transforms.build:transforms # So we can do "requires-level"
```

Comment:
Might be worth pulling `requires-level` processing into a separate transform?

Reply:
Note: even if it is in a separate transform, the build transforms could still import and use it implicitly.